### PR TITLE
Added community section

### PR DESCRIPTION
**Pull request by abbey-titcomb**
Added a Join our Community section for info on our community channels.

Screen Shot 2020-11-30 at 12 11 13 PM

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ desktop client.
 
 Radicle development is public by default and completely open-source. You can
 follow along development in our [Matrix channel][mc] or on
-[radicle.community][co].
+[radicle.community][co]. For more information on our community channels, see [Join our Community][cc].
```
Comment (xla) [inline docs/getting-started.md:14]:
```suggestion
[radicle.community][co]. For more information on our community channels, see
[Join our Community][cc].
```

File: docs/using-radicle/community.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: Join the Community
+---
+
+To get involved with the Radicle project, start by joining our community channels:
+
+### Join our [Matrix chat][mc]
+_For real-time discussion on everything related to Radicle_
+
+The best way to join our chat is by making an account on
+matrix.radicle.community and joining our [community homepage]. This is where you
```
Comment (rudolfs) [inline docs/using-radicle/community.md:12]:
```suggestion
matrix.radicle.community and joining our [community homepage][mc]. This is where you
```

File: docs/using-radicle/community.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+[mc]: https://matrix.radicle.community/
+[sp]: https://matrix.to/#/#support:radicle.community
+[gn]: https://matrix.to/#/#general:radicle.community
+[ch]: https://matrix.to/#/+home:radicle.community
+[cf]: https://radicle.community/
+[tl]: https://twitter.com/radicle
+[el]: http://eepurl.com/hhHxMX
+[tc]: https://t.me/joinchat/AAAAAFI8OTFiRiOWm8nTHg
+
```
Comment (rudolfs) [inline docs/using-radicle/community.md:41]:
```suggestion
```

**Review by xla — APPROVED** [review]
🍛 🏢 🕍 🛫

**Review by abbey-titcomb — COMMENTED** [review]
Fixed one link

**Pull request merged**